README.md

```diff
@@ -159,47 +194,61 @@ import Relude
 
 ## Difference from Prelude [↑](#structure-of-this-tutorial)
 
-* `head`, `tail`, `last`, `init` work with `NonEmpty a` instead of `[a]`.
-* `undefined` triggers a compiler warning, because you probably don't want to
-  leave `undefined` in your code. Either use `throwIO`, `Except`, `error` or
-  `bug`.
-* Multiple sorting functions are available without imports:
-  + `sortBy :: (a -> a -> Ordering) -> [a] -> [a]`: sorts list using given custom comparator.
-  + `sortWith :: Ord b => (a -> b) -> [a] -> [a]`: sorts a list based on some property of its elements.
-  + `sortOn :: Ord b => (a -> b) -> [a] -> [a]`: just like `sortWith`, but more
-    time-efficient if function is calculated slowly (though less
-    space-efficient). So you should write `sortOn length` (would sort elements
-    by length) but `sortWith fst` (would sort list of pairs by first element).
-* Functions `sum` and `product` are strict now, which makes them more efficient.
-* Since `show` doesn't come from `Show` anymore, you need to export
-  `Text.Show` module if you want to implement `Show` instance manually. This can be done like this:
-  ```haskell
-  import qualified Text.Show
-  ```
-* You can't call `elem` and `notElem` functions over `Set` and `HashSet`. These
-  functions are forbidden for these two types because of the performance reasons.
-* `error` takes `Text`.
-* `lookup` for lists is not exported.
+Main differences from `Prelude` can be grouped into the following categories:
+
+* Changed behavior of common functions
+  + `head`, `tail`, `last`, `init` work with `NonEmpty a` instead of `[a]`.
+  + `show` is polymorphic over return type.
+  + Functions `sum` and `product` are strict now, which makes them more efficient.
+  + You can't call `elem` and `notElem` functions over `Set` and `HashSet`.
+    These functions are forbidden for these two types because of the performance
+    reasons.
+  + `error` takes `Text`
+  + `undefined` triggers a compiler warning, because you probably don't want to
+    leave `undefined` in your code. Either use `throwIO`, `Except`, `error` or
+    `bug`.
+* Not reexported
+  + `read`
+  + `lookup` for lists
+  + `log`
+* Completely new functions are brougth into scope
+  + See [What's new?](#whats-new-) section for a detailed overview.
+* New reexports
+  + See [Reexports](#reexports-) section for a detailed overview.
 
 ## Reexports [↑](#structure-of-this-tutorial)
 
 `relude` reexports some parts of the following libraries:
 
 * `base`
-* `containers`
-* `unordered-containers`
-* `text`
 * `bytestring`
-* `transformers`
-* `mtl`
+* `containers`
 * `deepseq`
+* `hashable`
+* `mtl`
 * `stm`
+* `text`
+* `transformers`
+* `unordered-containers`
 
 If you want to clean up imports after switching to `relude`, you can use
 `relude`-specific [`.hlint.yaml`](.hlint.yaml) configuration for this task.
 
 ### base
 
+Multiple sorting functions are available without imports:
```

Comment:
without imports? probably this is an unnecessary phrase?

Reply:
I just copy-pasted this part. But I agree that part about imports in redundant in the context of preludes. Thanks for noticing!